CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Reduce memory usage by up to at least 30% with a full scrollback buffer
 - The number of zerowidth characters per cell is no longer limited to 5
 - `SpawnNewInstance` is now using the working directory of the terminal foreground process on macOS
+- Configuration `import` paths are now optional and will skip importing if path does not exist.
```

Comment:
The changelog entry should be in `0.7.0-dev` section. Also maybe something like that will work better.

```suggestion
- Providing non existing paths in `import` won't result in error and importing from these configs will be skipped
```